_includes/payment-resource.md

```diff
@@ -285,3 +285,4 @@ for the given operation.
 {% endcase %}
 
 [user-agent]: https://en.wikipedia.org/wiki/User_agent
+[merchant-identified-payer]: /payment-menu/other-features#merchant-identified-payer
```

Comment:
I think `payment-menu` in the URL here is going to cause problems in the other sections including `payment-resource.md`.